modules/basic-skills/src/views/full/slot.jsx

```diff
@@ -19,7 +19,8 @@ export class Slot extends React.Component {
     intents: [],
     actions: [],
     maxRetryAttempts: 3,
-    error: undefined
+    error: undefined,
+    turnExpiry: -1
```

Comment:
why -1 as default and not undefined ?

Reply:
Since it's a number type, it made validations easier (plus, it does make sense here as 'never expiring'